ios/StripeSdk.swift

```diff
@@ -778,6 +778,80 @@ class StripeSdk: RCTEventEmitter, STPApplePayContextDelegate, STPBankSelectionVi
         }
     }
     
+    @objc(verifyMicrodeposits:clientSecret:params:resolver:rejecter:)
+    func verifyMicrodeposits(
+        intentType: String,
+        clientSecret: String,
+        params: NSDictionary,
+        resolver resolve: @escaping RCTPromiseResolveBlock,
+        rejecter reject: @escaping RCTPromiseRejectBlock
+    ) -> Void {
+        let amounts = params["amounts"] as? NSArray
+        let descriptorCode = params["descriptorCode"] as? String
+        
+        if (amounts != nil && descriptorCode != nil || amounts == nil && descriptorCode == nil) {
+            resolve(Errors.createError("Failed", "You must provide either amounts OR descriptorCode, not both."))
+            return
+        }
+        
+        if let amounts = amounts {
+            if (amounts.count != 2) {
+                resolve(Errors.createError("Failed", "Expected 2 integers in the amounts array, but received " + String(amounts.count)))
```

Comment:
Will update these to not be just `"Failed"` strings and instead a Constant from Errors.swift